### PR TITLE
Awesome Books 

### PR DESCRIPTION
**Pull request by sumon766**
- Created Awesome Books.
>  ADD and Remove functionality is added in the script file.
>  data is stored in local storage

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    books = [];
+  } else {
+    books = JSON.parse(book);
+  }
+  books.push({
+    title,
+    auther,
+  });
+  localStorage.setItem('book', JSON.stringify(books));
+  displaybooks();
```
Comment (IngDavidTM) [inline script.js]: - [ ] While you did a good job adding books to your app :+1:, as a [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md) you should create a function to add new book with title and author parameters :100:

File: script.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+});
+
+function Remove(index) { // eslint-disable-line no-unused-vars
+  const book = localStorage.getItem('book');
+  books = JSON.parse(book);
+  books.splice(index, 1);
+  localStorage.setItem('book', JSON.stringify(books));
+  displaybooks();
+}
```
Comment (IngDavidTM) [inline script.js]: - [ ] You did a fantastic job storing all the books using LocalStorage :+1:, but I have noticed that when the user refreshes the page the content is not loading. To make it render you can call `displaybooks();` function at the end of your JavaScript file :+1: 

**Your project**

Screenshot from 2023-02-21 02-53-20

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @team 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @sumon766 & @Santosh-Konappanavar ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**